### PR TITLE
Fixes #11159: 

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/11159

### CODE REVIEW

File: rudder-web/src/main/resources/rudder-vhost-ssl.conf [hunk 1]
```diff
@@ -46,5 +46,12 @@
 
   # MSIE 7 and newer should be able to use keepalive
   BrowserMatch "MSIE [17-9]" ssl-unclean-shutdown
+  
+  #set "secure" and "HttpOnly" for all cookies without these pragma
+  Header edit Set-Cookie "(?i)^((?:(?!;\s?HttpOnly).)+)$" "$1; HttpOnly"
+  Header edit Set-Cookie "(?i)^((?:(?!;\s?secure).)+)$" "$1; secure"
```
Comment (fanf) [inline rudder-web/src/main/resources/rudder-vhost-ssl.conf]: Not sure if mod_header is enabled by default nor what should be done to ensure that.

Comment (peckpeck) [inline rudder-web/src/main/resources/rudder-vhost-ssl.conf]: nice negative search regex

**Comment by fanf** [comment]
OK, merging this PR

**Comment by fanf** [comment]
OK, merging this PR

**Pull request merged**